waterloop-site/src/App.tsx

```diff
@@ -100,12 +100,12 @@ class App extends React.Component<{}, State> {
             <Route path="/posting/:id">
               <JobPostingPage />
             </Route>
+            {/* <Route component={} />asd */}
```

Comment:
What we also want, is it to redirect to a /404 link. Refer to this link: https://reactrouter.com/web/api/Redirect